edsdme/blocks/announcements-preview/announcements-preview.css

```diff
@@ -0,0 +1,74 @@
+/* stylelint-disable */
+.announcement-item,
+.link-wrapper {
+  display: flex;
+  align-items: center;
+  margin: 26px 0px;
+  background-color: white;
+  width: 100%;
+  height: 130px;
+  max-width: 600px;
+}
+
+announcements-preview .con-button {
+  margin: 5px 0px;
+}
+
+.card-image {
+  min-width: 100px;
+  width: 20%;
+  height: 100%;
+  margin-right: 20px;
+  overflow: hidden;
+}
+
+.card-image picture,
+.card-image img {
+  width: 100%;
+  height: 100%;
+  object-fit: cover;
+}
+
+.card-content {
+  height: 100%;
+  width: 80%;
+  overflow: hidden;
+}
+
+.card-content .card-title {
+  margin: 25px 0px 15px 0px;
+  padding: 0;
+  line-height: 1.2;
+  overflow: hidden;
+  display: -webkit-box;
+  -webkit-box-orient: vertical;
+  -webkit-line-clamp: 1;
+  text-overflow: ellipsis;
+  color: black;
+  text-decoration: none;
+}
+
+.card-description {
+  margin: 10px 0px;
+  padding: 0;
+  line-height: 1.2;
+  overflow: hidden;
+  display: -webkit-box;
+  -webkit-box-orient: vertical;
+  color: #666;
```

Comment:
In general I think we can make better use of the spectrum CSS variables provided, perhaps `--color-gray-600` would do

https://main--milo--adobecom.hlx.live/libs/styles/styles.css